File: GOG.Models/AccountProduct.cs
using System.Runtime.Serialization;
using ProtoBuf;

using Models.ProductTypes;

namespace GOG.Models
{
    [ProtoContract, DataContract]
    public class AccountProduct: ProductCore
    {
        [ProtoMember(1),DataMember(Name = "isGalaxyCompatible")]
        public bool IsGalaxyCompatible { get; set; }
        [ProtoMember(2),DataMember(Name = "tags")]
        public string[] Tags { get; set; }
        [ProtoMember(3),DataMember(Name = "availability")]
        public Availability Availability { get; set; }
        [ProtoMember(4),DataMember(Name = "image")]
        public string Image { get; set; }
        [ProtoMember(5),DataMember(Name = "url")]
        public string Url { get; set; }
        [ProtoMember(6),DataMember(Name = "worksOn")]
        public WorksOn WorksOn { get; set; }
        [ProtoMember(7),DataMember(Name = "category")]
        public string Category { get; set; }
        [ProtoMember(8),DataMember(Name = "rating")]
        public int Rating { get; set; }
        [ProtoMember(9),DataMember(Name = "isComingSoon")]
        public bool IsComingSoon { get; set; }
        [ProtoMember(10),DataMember(Name = "isMovie")]
        public bool IsMovie { get; set; }
        [ProtoMember(11),DataMember(Name = "isGame")]
        public bool IsGame { get; set; }
        [ProtoMember(12),DataMember(Name = "slug")]
        public string Slug { get; set; }
        [ProtoMember(13),DataMember(Name = "updates")]
        public int Updates { get; set; }
        [ProtoMember(14),DataMember(Name = "isNew")]
        public bool IsNew { get; set; }
        [ProtoMember(15),DataMember(Name = "dlcCount")]
        public int DLCCount { get; set; }
        [ProtoMember(16),DataMember(Name = "releaseDate")]
        public TimezoneDate ReleaseDate { get; set; }
        [ProtoMember(17),DataMember(Name = "isBaseProductMissing")]
        public bool IsBaseProductMissing { get; set; }
        [ProtoMember(18),DataMember(Name = "isHidingDisabled")]
        public bool IsHidingDisabled { get; set; }
        [ProtoMember(19),DataMember(Name = "isInDevelopment")]
        public bool IsInDevelopment { get; set; }
        [ProtoMember(20),DataMember(Name = "isHidden")]
        public bool IsHidden { get; set; }
    }
}

File: internal/gog/remote/schema/accountProduct.go
// Copyright (c) 2020. All rights reserved.
// Use of this source code is governed by an MIT-style license that can be
// found in the LICENSE file.

package schema

type AccountProduct struct {
	IsGalaxyCompatible bool          `json:"isGalaxyCompatible"`
	Tags               []interface{} `json:"tags"`
	ID                 int           `json:"id" bson:"_id"`
	Availability       struct {
		IsAvailable          bool `json:"isAvailable"`
		IsAvailableInAccount bool `json:"isAvailableInAccount"`
	} `json:"availability"`
	Title   string `json:"title"`
	Image   string `json:"image"`
	URL     string `json:"url"`
	WorksOn struct {
		Windows bool `json:"Windows"`
		Mac     bool `json:"Mac"`
		Linux   bool `json:"Linux"`
	} `json:"worksOn"`
	Category     string `json:"category"`
	Rating       int    `json:"rating"`
	IsComingSoon bool   `json:"isComingSoon"`
	IsMovie      bool   `json:"isMovie"`
	IsGame       bool   `json:"isGame"`
	Slug         string `json:"slug"`
	Updates      int    `json:"updates"`
	IsNew        bool   `json:"isNew"`
	DlcCount     int    `json:"dlcCount"`
	ReleaseDate  struct {
		Date         string `json:"date"`
		TimezoneType int    `json:"timezone_type"`
		Timezone     string `json:"timezone"`
	} `json:"releaseDate"`
	IsBaseProductMissing bool `json:"isBaseProductMissing"`
	IsHidingDisabled     bool `json:"isHidingDisabled"`
	IsInDevelopment      bool `json:"isInDevelopment"`
	IsHidden             bool `json:"isHidden"`
}
